less/home-page.less

```diff
@@ -53,9 +84,39 @@
     display: flex;
     justify-content: space-between;
 
-    .destination {
-      width: 30%;
-      margin-bottom: 30px;
+
+    @media @mobile {
+      display: flex;
+      flex-wrap: nowrap;
+      flex-direction: column;
+      align-items: center;
+
+
     }
+
+
+
   }
-}// home
\ No newline at end of file
+
+
+  .destination {
+    width: 30%;
+    margin-bottom: 30px;
+
+
+
+    @media @mobile {
```

Comment:
Great job setting up media queries for mobile view! Looks like you're using the same flexbox properties on a lot of these, would be a great time to use a mixin in the future!